specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -9444,6 +9444,18 @@
         "primary": {
```

Comment:
Missing from this change is adding NetworkApiVersion to the VMSS netowrkProfile

Reply:
it is in VirtualMachineScaleSetNetworkProfile line 11459